internal/analytics/reporter.go

```diff
@@ -42,3 +42,11 @@ type ReportEvent struct {
 	PluginName            string
 	AnonymizedEventFields map[string]any
 }
+
+type ReportCommand struct {
```

Comment:
**nit** 
```suggestion
type ReportCommandInput struct {
```
while imported externally we have `reporter.ReportCommand` that has two meaning, and also it won't allow us to add `ReportCommandOutput` later if needed.

Reply:
I know, I wanted to do that but I went with consistency to the `ReportEvent` , as then I would rename it too... but ok, I will rename it 😄